src/parser/jobs/sam/modules/ReadyProcs.tsx

```diff
@@ -105,6 +147,6 @@ export class ReadyProcs extends CoreProcs {
 	override overwroteProcIcon = this.data.actions.GYOFU.icon
 	override overwroteProcContent =
 		<Trans id="sam.readyprocs.suggestions.overwrite.content">
-			Avoid overwriting your procs. Make sure you consume all procs from <DataLink action="MEIKYO_SHISUI"/> before using it again.
+			Avoid overwriting your procs.
```

Comment:
You should probably reset associated translations since there's now a missing DataLink in translations and part of the tip is now obsolete